core/src/main/java/cz/o2/proxima/repository/AttributeFamilyDescriptor.java

```diff
@@ -36,11 +37,70 @@
 import lombok.Getter;
 import lombok.Setter;
 import lombok.experimental.Accessors;
+import lombok.extern.slf4j.Slf4j;
 
 /** A family of attributes with the same storage. */
 @Evolving("Affected by #66")
+@Slf4j
 public class AttributeFamilyDescriptor implements Serializable {
 
+  public static final String CFG_CONSUMER_NAME_GENERATOR = "cfg.consumer.name.generator";
+  @Getter private final StorageType type;
+
+  public static Builder newBuilder() {
+    return new Builder();
+  }
+
+  @Getter private final String name;
+
+  @Getter private final EntityDescriptor entity;
+
+  @Getter private final URI storageUri;
+  /** Access type allowed to this family. */
+  @Getter private final AccessType access;
+
+  private final List<AttributeDescriptor<?>> attributes;
+  @Getter private final StorageFilter filter;
+  @Nullable private final String source;
+  @Getter private final AttributeFamilyConsumerNameGenerator consumerNameGenerator;
```

Comment:
This looks it should generate name of consumers consuming *from* this family. That is not the case, the generated name is used as name of consumer consuming *from* primary family and writing into *this* family.

Reply:
Changed